x/eibc/types/keys.go

```diff
@@ -34,8 +34,6 @@ var (
 	PendingDemandOrderKeyPrefix = []byte{0x00, 0x01}
 	// FinalizedDemandOrderKeyPrefix is the prefix for finalized demand orders
 	FinalizedDemandOrderKeyPrefix = []byte{0x00, 0x02}
-	// RevertedDemandOrderKeyPrefix is the prefix for reverted demand orders
-	RevertedDemandOrderKeyPrefix = []byte{0x00, 0x03}
 )
```

Comment:
IMO it's simpler to err on side of caution here and reserve `_ = []byte{..}` rather than deleting